### ISSUE TITLE
Стр: ЛК Админа Отчеты продаж

### ISSUE THREAD

**Issue by MrMatRog** [software-sales-and-installations/SoftPlat2048#40]
Требования расписаны в карточке #48

**Comment by Wereflyers** [comment]
Для mvp отчеты как список на сайте, для 2 релиза будут таблицы

**Comment by MrMatRog** [comment]
@Wereflyers Да, именно так и предполагалось. Знаю, что бэки часть свою сделали, фронты же пока не сделали часть, которую предполагали на MVP, поэтому им необходимо сделать и то и другое. Задачу разделю на веб-отчет и выгрузку отчета. Спасибо!
